### DISCUSSION TITLE
would like to know how to implement the database transaction.

### DISCUSSION THREAD

**Discussion by gonta1026** [supabase/supabase#12276] [General]
## consultation

I am currently using the supabase client to create and update for multiple tables, but i know that SUPABASE cannot implement client-side transactions.

What is the process and how should we implement it in supabase to handle this?

Any sample code or documentation you can share would be appreciated.

※If you don't mind, it would be helpful if you could provide an example where each record in one todos table can have multiple records in the comments table.

**Comment by GaryAustin1** [comment]
If you use an rpc call to a postgres function all the code in the function will be handled as a transaction.
